### PR TITLE
Contributor Guidelines

### PR DESCRIPTION
**Pull request by iamrecursion**
Added CONTRIBUTING.md

Edited README.md to reformat build guidelines and add license section.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -49,28 +70,90 @@ The prerequisites for building Luna Studio are the following (with Ubuntu apt pa
...
+
+## Contributing to Luna Studio
+If you are interested in contributing to the development of Luna, please read
+the 
+[`CONTRIBUTING.md`](https://github.com/luna/luna-studio/blob/master/CONTRIBUTING.md)
+file. 
+
+## License
+This repository is licensed under the
+[GNU GPL 3.0](https://opensource.org/licenses/GPL-3.0), as specified in the
```
Comment (mikusp) [inline README.md]: luna-studio repo is licensed under AGPL 3.0

File: README.md [hunk 2]
```diff
@@ -49,28 +70,90 @@ The prerequisites for building Luna Studio are the following (with Ubuntu apt pa
...
+
+Any work intentionally submitted for inclusion in Luna shall be licensed under
+this CLA.
+
+The CLA you sign applies to all repositories associated with the Luna project 
+([Luna](https://github.com/luna/luna), 
+[Luna RFCs](https://github.com/luna/luna-rfcs), etc), so you will only have 
+to sign it once at the start of your contributions. 
+
+## License
```
Comment (mikusp) [inline README.md]: I think this section belongs to luna repo

**Comment by kustosz** [comment]
@mikusp good catch! Since this already got merged after the previous reviewer (yeah, okay, that was me) missed all these, I've fixed it on master right away.

**Comment by iamrecursion** [comment]
Thanks @mikusp. This is why two reviewers are important...

**Pull request merged**